index.html

```diff
@@ -0,0 +1,180 @@
+        <div class="date-time">
+          <h3>2023.07.05(WED) ~ 06(THU)</h3>
+          <p>@ iHub, Senteu Plaza Nairobi</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="main-programs">
+      <h2>Our Programs</h2>
+      <div class="program-cards">
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/react-2.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>React</h4>
+          </div>
+          <div class="logo-description">
+            <p>React is a free JavaScript library for building user interfaces based on components.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/rails-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Ruby on Rails</h4>
+          </div>
+          <div class="logo-description">
+            <p>Rails is a framework that provides default structures for databases.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/devops-2 (1).svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Devops</h4>
+          </div>
+          <div class="logo-description">
+            <p>DevOps improves and shortens the systems development life cycle.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/kotlin-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Kotlin</h4>
+          </div>
+          <div class="logo-description">
+            <p>Kotlin is a cross-platform high-level programming language with type inference.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/vue-9.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Vue.js</h4>
+          </div>
+          <div class="logo-description">
+            <p>Vue.js is a JavaScript framework for building user interfaces and single-page applications.</p>
+          </div>
+        </div>
+      </div>
+      <a href="#">Join Our Program</a>
+    </section>
+
+    <section class="speakers">
+      <h2>Featured Speakers</h2>
+      <div id="speaker-cards"></div>
+    </section>
+
+    <section class="partners">
+      <h2>Partners</h2>
+      <div class="partner-logos">
+        <img src="./images/apple-13.svg" alt="">
+        <img src="./images/google-1-1.svg" alt="">
+        <img src="./images/meta-1.svg" alt="">
+        <img src="./images/microsoft.svg" alt="">
+        <img src="./images/oracle-6.svg" alt="">
+      </div>
+    </section>
```

Comment:
- [ ] Please make sure that your sponsor section has the same design( Height, spacing) as the original design. Also, make sure that your images have the same opacity and size as on the original design in order to make your website design wonderful 👍 

#### Your design
![sponsors](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/12ea64ab-f94d-4696-8ba2-6f2fc587ad2c)

#### The original design
![sponsors](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/19f00143-9455-462c-8076-779ae72b1d48)

Reply:
Fixed :+1: